kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/storage/StorageWriteApiDefaultStream.java

```diff
@@ -104,7 +114,30 @@ JsonStreamWriter getDefaultStream(TableName table, List<ConvertedRecord> rows) {
       StorageWriteApiRetryHandler retryHandler = new StorageWriteApiRetryHandler(table, getSinkRecords(rows), retry, retryWait, time);
       do {
         try {
-          return JsonStreamWriter.newBuilder(t, getWriteClient()).build();
+          BigQueryWriteClient writeClient = getWriteClient();
+
+          // Copied from JsonStreamWriter::newBuilder
+          // TODO: Extract logic into superclass and leverage in StorageWriteApiBatchApplicationStream
```

Comment:
Should we address? Otherwise, this feature only works for the default stream, right?